lib/membrane_h264_plugin/parser/nalu.ex

```diff
@@ -22,15 +22,10 @@ defmodule Membrane.H264.Parser.NALu do
           prefix_length: pos_integer(),
           type: Membrane.H264.Parser.NALuTypes.nalu_type(),
           payload: binary(),
-          status: :valid | :error
+          status: :valid | :error,
+          timestamps: {pts :: integer() | nil, dts :: integer() | nil}
```

Comment:
How about making some type `@type timestamps :: {pts :: integer() | nil, dts :: integer() | nil}` and use it here and in `nalu_parser.ex`?